src/TMedia.cpp

```diff
@@ -110,33 +110,33 @@ void TMedia::stopMedia(TMediaData& mediaData)
     QList<TMediaPlayer> mTMediaPlayerList;
 
     switch (mediaData.getMediaProtocol()) {
-        case TMediaData::MediaProtocolMSP:
-            switch (mediaData.getMediaType()) {
-                case TMediaData::MediaTypeSound:
-                    mTMediaPlayerList = mMSPSoundList;
-                    break;
-                case TMediaData::MediaTypeMusic:
-                    mTMediaPlayerList = mMSPMusicList;
-                    break;
-            }
+    case TMediaData::MediaProtocolMSP:
```

Comment:
For the formatting - format it using the .clang-format file we've already got. I don't know if VS supports it, hopefully there's a plugin - both CLion and Qt Creator do. If there isn't, there's a command line app you can run.

Much easier to do consistent formatting when everyone has access to it!

Reply:
Thanks Vadi!  There was a way and I found the file in Mudlet.

Reply:
@vadi2 what news from IRE? :)

I executed a `lua gmcp.Client` on Imperian and did not see a
```
{
  Media = {
    url = "https://www.imperian.com/.../"
  }
```
... so it is less likely they would be impacted as that's typically something that would be done at login.